conductr_cli/constants.py

```diff
@@ -1,14 +1,19 @@
 import os
 
-# Preload this much data in the bndl tool to determine input type of a stream
-BNDL_PEEK_SIZE = 16384
+# Preload this much data in the bndl tool to determine input type of a stream. Since we use this to determine
+# if a stream is a plain bundle conf, this should be set to hold the maximum size of a reasonable bundle.conf
+# or else auto-detection will fail.
+BNDL_PEEK_SIZE = 65536
 
 # Defaults for creating bundles without specifying params
+BNDL_DEFAULT_ANNOTATIONS = {}
 BNDL_DEFAULT_COMPATIBILITY_VERSION = '0'
 BNDL_DEFAULT_DISK_SPACE = 1073741824
 BNDL_DEFAULT_MEMORY = 402653184
+BNDL_DEFAULT_NAME = 'bundle'
 BNDL_DEFAULT_NR_OF_CPUS = 0.1
 BNDL_DEFAULT_ROLES = []
```

Comment:
I think we have `web` as our default roles in SBT ConductR:

https://github.com/typesafehub/sbt-conductr/blob/master/src/main/scala/com/lightbend/conductr/sbt/BundlePlugin.scala#L49

Not sure whether we want to keep it consistent with `sbt-conductr` - perhaps @huntc can confirm here.